Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># SAE3.01\r\nBonjour, vous êtes ici sur le projet de la création d'un site web pour gérer un tournoi\r\nde chôlage. Ce projet est réalisé par l'Informa-Team dans le cadre de la SAE 3.01 du BUT \r\nInformatique. Si vous n'avez rien à voir avec cette SAE, je vous prierai de ne pas toucher\r\nà quoi que ce soit et quitter la page. Merci.\r\n\r\n- Chef de projet: Michel Ewan\r\n- Sous-Chef de projet: Meriaux Thomas  \r\nAccompagnés de:\r\n- Monsieur Benredouane Iliès\r\n- Monsieur Hostelart Anthony\r\n- Monsieur Faës Hugo\r\n\r\n\r\n \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision a1f0b300cca960a31fe8f978f499a827493740c2)
+++ b/README.md	(date 1699600251637)
@@ -2,7 +2,7 @@
 Bonjour, vous êtes ici sur le projet de la création d'un site web pour gérer un tournoi
 de chôlage. Ce projet est réalisé par l'Informa-Team dans le cadre de la SAE 3.01 du BUT 
 Informatique. Si vous n'avez rien à voir avec cette SAE, je vous prierai de ne pas toucher
-à quoi que ce soit et quitter la page. Merci.
+à quoi que ce soit et de quitter la page. Merci.
 
 - Chef de projet: Michel Ewan
 - Sous-Chef de projet: Meriaux Thomas  
