Index: Class/Member.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\nclass Member {\r\n    public $username;\r\n    private $mail;\r\n    private $name;\r\n    private $firstname;\r\n    private $birthday;\r\n    private $password;\r\n    private $isRegistering;\r\n\r\n    function __construct($un, $m, $n, $fn, $b, $p, $ir) {\r\n        $this->username = $un;\r\n        $this->mail = $m;\r\n        $this->name = $n;\r\n        $this->firstname = $fn;\r\n        $this->birthday = $b;\r\n        $this->password = $p;\r\n        $this->isRegistering = $ir;\r\n    }\r\n}\r\n
===================================================================
diff --git a/Class/Member.php b/Class/Member.php
--- a/Class/Member.php	
+++ b/Class/Member.php	
@@ -18,4 +18,46 @@
         $this->password = $p;
         $this->isRegistering = $ir;
     }
+
+    public function getBirthday()
+    {
+        return $this->birthday;
+    }
+
+    public function getFirstname()
+    {
+        return $this->firstname;
+    }
+
+    public function getIsRegistering()
+    {
+        return $this->isRegistering;
+    }
+
+    public function getMail()
+    {
+        return $this->mail;
+    }
+
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    public function getPassword()
+    {
+        return $this->password;
+    }
+
+    function wantBecomePlayer() {
+        $this->isRegistering = true;
+        /*mail($this->mail, "Inscription", "Bonjour,`\n Vous avez soumis votre inscription
+        pour le tournoi de Chôlage. Nos administrateurs vont vérifier vos informations et valider ou réfuter votre 
+        inscription, le cas échéant, vous recevrez un mail de confirmation. Nous vous remercions de votre inscription et 
+        espérons vous recevoir très vite pour l'événement. \n\n Cordialement,\nL'équipe du tournoi !!!");*/
+    }
+
+    public function setIsRegistering() {
+        $this->isRegistering = !$this->isRegistering;
+    }
 }
